pkg/filter/http/apiconfig/api_config.go

```diff
@@ -116,14 +116,16 @@ func (f *Filter) Decode(ctx *contexthttp.HttpContext) filter.FilterStatus {
 	req := ctx.Request
 	v, err := f.apiService.MatchAPI(req.URL.Path, fc.HTTPVerb(req.Method))
 	if err != nil {
-		ctx.SendLocalReply(http.StatusNotFound, constant.Default404Body)
+		errResp := contexthttp.APINotFound.New()
+		ctx.SendLocalReply(errResp.Status, errResp.ToJSON())
 		e := errors.Errorf("Requested URL %s not found", req.URL.Path)
 		logger.Debug(e.Error())
 		return filter.Stop
 	}
 
 	if !v.Enable {
-		ctx.SendLocalReply(http.StatusNotAcceptable, constant.Default406Body)
+		errResp := contexthttp.NotAcceptable.WithError(fmt.Errorf("API not online"))
```

Comment:
if the format func is not used when creating a new err, use `errors.New` instead of `fmt.Errorf`. `fmt.Errorf` only used when need format

```suggestion
		errResp := contexthttp.NotAcceptable.WithError(errors.New("API not online"))
```